src/core/network-openssl.c

```diff
@@ -601,18 +601,19 @@ static void set_cipher_info(TLS_REC *tls, SSL *ssl)
 	tls_rec_set_cipher_size(tls, SSL_get_cipher_bits(ssl, NULL));
 }
 
-static void set_pubkey_info(TLS_REC *tls, X509 *cert, unsigned char *cert_fingerprint, size_t cert_fingerprint_size, unsigned char *public_key_fingerprint, size_t public_key_fingerprint_size)
+static gboolean set_pubkey_info(TLS_REC *tls, X509 *cert, unsigned char *cert_fingerprint, size_t cert_fingerprint_size, unsigned char *public_key_fingerprint, size_t public_key_fingerprint_size)
 {
+	int ret = 1;
```

Comment:
Might as well set it as gboolean and TRUE (which is the exact same type and value in practice, but yeah)

Reply:
Done in https://github.com/irssi/irssi/pull/1036/commits/a6ade0bc96e32a14cc864cbe0e143e043fc8583a.